### PR TITLE
feat(components/tabs): add keyboard navigation to vertical tabset

### PR DESCRIPTION
**Pull request by Blackbaud-PaulCrowder**
[AB#2599941](https://dev.azure.com/blackbaud/f565481a-7bc9-4083-95d5-4f953da6d499/_workitems/edit/2599941)

### CODE REVIEW

**Comment by blackbaud-sky-build-user** [comment]
[Storybook preview](https://blackbaud.github.io/skyux-pr-preview/1978/storybook/)

Component Storybooks:

- [action-bars-storybook](https://blackbaud.github.io/skyux-pr-preview/1978/storybooks/action-bars-storybook/)
- [pages-storybook](https://blackbaud.github.io/skyux-pr-preview/1978/storybooks/pages-storybook/)
- [split-view-storybook](https://blackbaud.github.io/skyux-pr-preview/1978/storybooks/split-view-storybook/)
- [tabs-storybook](https://blackbaud.github.io/skyux-pr-preview/1978/storybooks/tabs-storybook/)
- [text-editor-storybook](https://blackbaud.github.io/skyux-pr-preview/1978/storybooks/text-editor-storybook/)

Apps:

- [code-examples](https://blackbaud.github.io/skyux-pr-preview/1978/code-examples/)
- [dep-graph](https://blackbaud.github.io/skyux-pr-preview/1978/dep-graph/)
- [integration](https://blackbaud.github.io/skyux-pr-preview/1978/integration/)
- [playground](https://blackbaud.github.io/skyux-pr-preview/1978/playground/)

File: libs/components/tabs/src/lib/modules/vertical-tabset/vertical-tabset-group.component.html [hunk 1]
```diff
@@ -3,21 +3,28 @@
   <div
     class="sky-vertical-tabset-group-header"
     [ngClass]="{
-      'sky-vertical-tabset-group-header-sub-open sky-font-emphasized':
-        subMenuOpen(),
+      'sky-vertical-tabset-group-header-active sky-font-emphasized':
+        isActive(),
```
Comment (Blackbaud-ErikaMcVey) [inline libs/components/tabs/src/lib/modules/vertical-tabset/vertical-tabset-group.component.html:7]: is this the only way to keep track if the tab is active? Won't it get called every time change detection is run?

**Review by Blackbaud-ErikaMcVey — APPROVED** [review]
one small comment but it looks like that pre-existed this PR so probably out of scope

File: libs/components/tabs/src/lib/modules/vertical-tabset/vertical-tabset-group.component.html (hunk 1 above)
Comment (Blackbaud-PaulCrowder) [reply libs/components/tabs/src/lib/modules/vertical-tabset/vertical-tabset-group.component.html:7]: Yeah, it's not ideal, but I decided not to refactor it, just rename it to reflect what it actually does.

**Comment by blackbaud-sky-build-user** [comment]
Cherry-pick successful! :cherries: :tada: See https://github.com/blackbaud/skyux/pull/1986

**Pull request merged**